nfeliciano/mangiamo
# Information modal

This branch moves the User Information Div into a modal in the main page.  This change is needed because the redirect back to the login page is unnecessary, and needlessly confusing.  This also allows for more streamline code when it comes to Facebook integration.  Kevin is going to add the facebook API calls (after permissions are gained) to pre-populate some of the data.

Testing this Pull Request:
1. Log out of the application
2. Delete yourself from the database (don't want to lose your friends?  Then you cannot test this pull request)
3. Log in with Facebook
4. Try to Create or Join a meal, and you will be met with the new User Information Modal
5. If you exit the process early, such as hitting the "x" or clicking outside the modal, nothing is saved and the same modal will come up again if you try to Create or Join a meal
6. Complete the information necessary, and hit "Submit".  A secondary modal will appear with a helpful message to the user telling them what has happened, and what they can do now (automatically adding the meal they just tried to create is not something I was able to do.)
7. You can now Create and Join meals as per desired
